sendContactUsEmail.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+session_start();//create a session for each user and use the session states
+
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+
+require 'PHPMailer/src/Exception.php';
+require 'PHPMailer/src/PHPMailer.php'; //Import PHPMailer
+require 'PHPMailer/src/SMTP.php';
+
+if(isset($_POST['submitContact']))
+{
+
+        //use htmlspecialchars to prevent cross-site scripting and propper handling of special characters
+        $yourname = htmlspecialchars($_POST['yourname']);
+        $email = htmlspecialchars($_POST['email']);
+        $phonenumber = htmlspecialchars($_POST['phonenumber']);;
+        $message= htmlspecialchars($_POST['message']);
+
+        $requestCallback =  isset($_POST['requestCallback']) ? 'Yes' : 'No';
+       
+        // Do your email processing here, such as sending the email
+            
+        //Create an instance; passing `true` enables exceptions
+            $mail = new PHPMailer(true);
+
+            try {
+                //Server settings
+                //$mail->SMTPDebug = SMTP::DEBUG_SERVER;                            //Enable verbose debug output
+                $mail->isSMTP();                                                   //Send using SMTP
+                $mail->SMTPAuth   = true;                                         //Enable SMTP authentication
+
+                $mail->Host       = 'mail.techgirlshub.co.za';                    //Set the SMTP server to send through
+                $mail->Username   = 'testaccount3@techgirlshub.co.za';            //SMTP username        
```

Comment:
change this and use admin@techgirlshub.co.za, password : Root96void23.(include the fullstop in the password)